modules/ringieraxelspringerBidAdapter.js

```diff
@@ -314,9 +315,9 @@ const parseAuctionConfigs = (serverResponse, bidRequest) => {
     auctionConfigs.push({
       'bidId': bid.bidId,
       'config': {
-        'seller': 'https://csr.onet.pl',
-        'decisionLogicUrl': `https://csr.onet.pl/${encodeURIComponent(bid.params.network)}/v1/protected-audience-api/decision-logic.js`,
-        'interestGroupBuyers': ['https://csr.onet.pl'],
+        'seller': BO_CSR_ONET,
```

Comment:
is this the only buyer used at the moment? (grepping seems to suggest yes).

I understand the intent and in the long run it may make sense (a bit early to tell) but currently, the effect of this PR would be to transform this single string into a much bigger library from which we use this single string.

Should we start by only including buyers that we know are in use? or wait until there's more than one consumer for the library?

Reply:
Yes, I think so, the expectation is the buyers are only needed to be defined in a future case

Reply:
Update, I commented out the unused imports